docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -146,42 +137,32 @@ Here is an example targeting PCL Profile 328. Profile 328 supports: .NET Standar
 
 You can now build.
 
-Using DNX:
-
-```
-$ dnu restore
-$ dnu build
-```
-
-Using .NET CLI:
-
 ```
 $ dotnet restore
-$ dotnet compile
+$ dotnet build
 ```
 
-You will now notice two new entries in your `/bin/Debug` folder:
+You will now notice the following entry in the `/bin/Debug` folder:
 
 ```
-/bin
-   /Debug
-      /dotnet55
-      /portable-net40+sl50+netcore45+wpa81+wp8
+$ ls bin/Debug
+
+portable-net40+sl50+netcore45+wpa81+wp8/
 ```
 
-Each folder contains distinct `.dll` files corresponding to their named target.
+This folder contains the `.dll` files necessary to run your library.
 
-## How to cross-compile for .NET Core and .NET Framework
+## How to Multitarget
 
-**NOTE:** These instructions assume you have .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+**NOTE:** These instructions assume you have the .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
 
-You may need to support older versions of .NET Framework when supporting both .NET Framework and .NET Core.  You may also wish to use newer APIs and language constructs for newer targets.  You can do this by using `#if` directives.
+You may need to support older versions of the .NET Framework when supporting both the .NET Framework and .NET Core.  You may also wish to use newer APIs and language constructs for newer targets.  You can do this by using `#if` directives.
```

Comment:
I like that a lot.  Changed just a few small words, but I'll happily take this 😄